packages/web/public/static/locales/de/common.json

```diff
@@ -114,7 +114,8 @@
       "threadCommentThanks": "hat für einen Feedback-Kommentar Danke gesagt",
       "threadCommentThanks_plural": "hat für {{count}} Feedback-Kommentare Danke gesagt",
       "newPosts": "Dieser Journaler, dem du folgst, hat einen neuen Beitrag veröffentlicht",
-      "newPosts_plural": "Journaler, denen du folgst, haben {{count}} neue Beiträge veröffentlicht"
+      "newPosts_plural": "Journaler, denen du folgst, haben {{count}} neue Beiträge veröffentlicht",
+      "mention": "{{mentioner}} mentioned you in a comment"
```

Comment:
@LindasLinguas just letting you know this new translation string is here in case you have a few minutes to help us translate it! 🙏🏼😄 No worries at all if not - we're planning on wrapping this up in 1 week 🚀